get_weather.py

```diff
@@ -0,0 +1,17 @@
+from decimal import *
+from datetime import date, timedelta, datetime
+import formatting_Utils
+import get_weather_Utils
+
```

Comment:
i think you do not need so many comments

Reply:
Alright. Was just for my reviewers to get into it easier.